setup.cfg

```diff
@@ -0,0 +1,7 @@
+[versioneer]
+VCS = git
+style = pep440
+versionfile_source = pyfftw/_version.py
+versionfile_build = pyfftw/_version.py
+tag_prefix =
```

Comment:
Good catch.  I have updated the PR accordingly.  `pyfftw.__version__` now reports `'0.10.4+66.g8e72f05'` in this branch